playwright/_impl/_transport.py

```diff
@@ -96,14 +96,19 @@ async def run(self) -> None:
         await super().run()
         self._stopped_future: asyncio.Future = asyncio.Future()
 
-        self._proc = proc = await asyncio.create_subprocess_exec(
-            str(self._driver_executable),
-            "run-driver",
-            stdin=asyncio.subprocess.PIPE,
-            stdout=asyncio.subprocess.PIPE,
-            stderr=_get_stderr_fileno(),
-            limit=32768,
-        )
+        try:
+            self._proc = proc = await asyncio.create_subprocess_exec(
+                str(self._driver_executable),
+                "run-driver",
+                stdin=asyncio.subprocess.PIPE,
+                stdout=asyncio.subprocess.PIPE,
+                stderr=_get_stderr_fileno(),
+                limit=32768,
+            )
+        except FileNotFoundError:
```

Comment:
In what case do you think this exception would be raised because on PyPI only wheels are uploaded and there's no chance of not having playwright driver installed ?

Reply:
first time if you clone the code from github, doesn't install the driver, run the pytest command and it hangs.

Reply:
If you want to install from github repo this is what you should be really doing: 
```console
$ pip install git+https://github.com/microsoft/playwright-python
```

Reply:
I want to fix the issue, so I need to test the source code and find the bug.

Reply:
Then, You can suggest to run `python setup.py bdist_wheel` which would fix the issue rather than just saying that it is not there

Reply:
yes! it's my mistake.

Reply:
You can read the contributing guide for some guidance on how to get everything setup for working on the code  https://github.com/microsoft/playwright-python/blob/master/CONTRIBUTING.md